src/select.coffee

```diff
@@ -400,6 +400,9 @@ Ember.AddeparMixins.ResizeHandlerMixin, Ember.Widgets.KeyboardHelper,
     )
   .property 'groupedContent.[]'
 
+  # whether or not to ensure that highlighted option is visible
```

Comment:
i think "ensureVisible" is not really that clear to begin with.  naming the thing that controls when that is triggered is a step away and even less clear.